### DISCUSSION TITLE
Guage component

### DISCUSSION THREAD

**Discussion by shivamBentley** [iTwin/iTwinUI-react#959] [Q&A]
I want a Gauge component as shown below in the image. Is there any Guage component available in the iTwinUi react library? if not, could you please suggest which Guage component should I use?

![Gauge](https://user-images.githubusercontent.com/110664877/204992196-dcc9768d-d8d1-4c94-a53a-9c7dac33a773.png)

**Comment by mayank99** [comment]
Hi @shivamBentley, there is nothing like this in iTwinUI and it seems like a very specific requirement that might not belong in our library. I would suggest building your own (it looks like a pretty simple one so should not be too much code) or using a third-party component if you can find one.
